wcfsetup/install/files/lib/system/database/Database.class.php

```diff
@@ -484,7 +484,7 @@ public function getDatabaseName()
     /**
      * Returns the name of the database user.
      *
-     * @return  string      user name
+     * @return  string      username
```

Comment:
As an example: This can be dropped and should likely be replaced by a proper return type.